include/kernel_includes.h

```diff
@@ -33,7 +33,6 @@
 #include <mempool_sys.h>
 #endif
 #include <kernel_version.h>
-#include <random/rand32.h>
```

Comment:
why are you taking the header include out of kernel_includes?

Reply:
nevermind, I looked at the commit message where it is described... I suspect you ran into this when adding the ctrdbg stuff... ;-)

Reply:
nope, the initial patch had this problem. rand32.h now includes an auto-generated file that is not available for unit tests.